docs/components/receive.md

```diff
@@ -66,6 +74,13 @@ The example content of `hashring.json`:
 With such configuration any receive listens for remote write on `<ip>10908/api/v1/receive` and will forward to correct one in hashring if needed
 for tenancy and replication.
 
+### Enabling Routing and Ingestion for the Thanos Receiver
+
+We have not added another flag for controlling this behaviour. Instead, we have relied on the existing flags and the below convention to enable and disable the ingestion and routing functionalities:
+* **Ingestion and Routing** - Make sure that `--receive.local-endpoint` and either `--receive.hashrings` or `--receive.hashrings-file` are provided.
+* **Ingestion** - Since routing behaviour is not needed, provide `--receive.local-endpoint` only.
```

Comment:
Since `--receive.local-endpoint` is only used to figure out if the current instance is in the specified hashring, it really doesn't make sense to the user to specify `--receive.local-endpoint` without also `--receive.hashring` too.

In my opinion it is much clearer for the user if we advise them to enable ingestion mode without specifying any extra parameters.

Also - my preferences would re-arrange these bullets so that number of extra parameters are increasing as the list goes down:
* Ingestion - no parameters.
* Routing - hashring parameters
* Ingestion & Routing - hashrigh parameters & local endpoint parameter.

Reply:
So do you propose to have a default value for `--receive.local-endpoint`, as it was earlier - [link](https://github.com/thanos-io/thanos/blob/release-0.16/cmd/thanos/receive.go#L122)?